flax/core/lift.py

```diff
@@ -149,21 +149,21 @@ def repack(inner_scope_tree):
         inner_scope.invalidate()
     for scope, out_variable_groups in zip(scopes, out_variable_groups_xs):
       for out_variable_group in out_variable_groups:
-        for kind, kind_variables in out_variable_group.items():
-          for name, value in kind_variables.items():
-            scope.put_variable(kind, name, value)
+        for col_name, collection in out_variable_group.items():
+          for name, value in collection.items():
+            scope.put_variable(col_name, name, value)
     return y
   return wrapper
 
 id_fn = lambda x: x
 
 def transform_module(fn: Callable[..., Any],
-                     target: KindFilter = 'param',
```

Comment:
Should we also considering making the "standard" collections into constance, e.g. `flax.linen.PARAMS = 'params'` and `flax.linen.normalization.BATCH_STATS = 'batch_stats'`?

Reply:
probably will do in seperate PR though.